eras/conway/impl/src/Cardano/Ledger/Conway/Delegation/Certificates.hs

```diff
@@ -1,23 +1,32 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
For consistency sake it would be good to move this pragma below the language extensions